libs/src/services/identity/index.js

```diff
@@ -31,6 +31,14 @@ class IdentityService {
     })
   }
 
+  async getUserEvents (walletAddress) {
```

Comment:
confirming this doesn't throw when an error is thrown for a user w/o entry in UserEvents?

Reply:
for any new users added after this migration they won't have an entry in UserEvents and will fail that check

Reply:
yeah -- success response {} is returned, which basically means needsRecoveryEmail = false